packages/apps/plugins/plugin-presenter/src/PresenterPlugin.tsx

```diff
@@ -22,6 +23,11 @@ type PresenterState = {
   presenting: boolean;
 };
 
+const isLocalMarkdownContent = (data: unknown): data is { content: EditorModel } =>
```

Comment:
I usually don't bother with exact types and do `data: any` in the parameters so that the code below is easier to read